ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=update
+VELOX_REPO=https://github.com/JkSelf/velox.git
+VELOX_BRANCH=write-metrics
```

Comment:
Do we need to include corresponding change to OAP Velox?

Reply:
Yes. I will pick this change to oap velox later if no further comment. Thanks.